packages/form-js-editor/src/render/hooks/useEffectDebugger.js

```diff
@@ -0,0 +1,59 @@
+const { useEffect, useRef, useCallback } = require('preact/hooks');
+
+const usePrevious = (value, initialValue) => {
+  const ref = useRef(initialValue);
+  useEffect(() => {
+    ref.current = value;
+  });
+  return ref.current;
+};
+
+export function useEffectDebugger(effectHook, dependencies, dependencyNames = [], effectName = 'noname') {
```

Comment:
What is this? Should we move this to our testing utils? Or remove it completely?